### PR TITLE
Introduces mailcatcher into development mode

### PR DESCRIPTION
**Pull request by hennevogel**
https://mailcatcher.me/

mailcatcher screenshot

### CODE REVIEW

File: src/api/db/migrate/20180221175514_change_configuration_obs_url_default.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ChangeConfigurationObsUrlDefault < ActiveRecord::Migration[5.1]
+  def change
+    change_column_default(:configurations, :obs_url, from: NULL, to: 'https://unconfigured.openbuildservice.org')
```
Comment (ChrisBr) [inline src/api/db/migrate/20180221175514_change_configuration_obs_url_default.rb]: Hm, is this really necessary?

Comment (hennevogel) [reply src/api/db/migrate/20180221175514_change_configuration_obs_url_default.rb]: What do you mean?

Comment (hennevogel) [reply src/api/db/migrate/20180221175514_change_configuration_obs_url_default.rb]: It's mentioned in the commit message of 7c27abd4988c0f93fd1b1deb3dce1862ff87a9ae

**Comment by hennevogel** [comment]
codecov hangs again. merging anyway as this doesn't touch code...

**Comment by hennevogel** [comment]
@openSUSE/open-build-service you need to pull the docker `frontend-base` image again as this changes the installed packages

**Pull request merged**